Objects/Objects/BuiltElements/Duct.cs

```diff
@@ -107,7 +109,7 @@ public class RevitDuct : Duct
     /// <remarks>Assign units when using this constructor due to <paramref name="width"/>, <paramref name="height"/>, and <paramref name="diameter"/> params</remarks>
     [SchemaInfo("RevitDuct", "Creates a Revit duct", "Revit", "MEP")]
     [SchemaDeprecated]
-    public RevitDuct(string family, string type, [SchemaMainParam] Line baseLine, string systemName, string systemType, Level level, double width, double height, double diameter, double velocity = 0, List<Parameter> parameters = null)
+    public RevitDuct(string family, string type, [SchemaMainParam] Line baseLine, string systemName, string systemType, Level level, double width, double height, double diameter, double velocity = 0, List<Parameter> parameters = null, Connector[] connectors = null)
```

Comment:
A new constructor needs to be added with the `connectors` argument to not break backwards compatibility with existing GH components